inc/modules/import/epub/class-epub201.php

```diff
@@ -554,4 +580,41 @@ protected function kneadHref( \DOMDocument $doc, $type, $href ) {
 
 		return $doc;
 	}
+
+
+		/**
+		 * Parse manifest with type 'application/xhtml+xml' to array
+		 *
+		 * @param \SimpleXMLElement $xml
+		 * @return array()
+		 */
+	protected function parseManifestToArray( \SimpleXMLElement $xml ) {
+
+		foreach ( $xml->manifest->children() as $item ) {
+			/** @var \SimpleXMLElement $item */
+			// Get attributes
+			$id = $title = $type = $href = '';
+			foreach ( $item->attributes() as $key => $val ) {
+				if ( 'id' === $key ) {
+					$id = (string) $val;
+				} elseif ( 'media-type' === $key ) {
+						$type = (string) $val;
+				} elseif ( 'href' === $key ) {
+						$href = $val;
+				}
+			}
+
+						// Skip
+			if ( 'application/xhtml+xml' !== $type ) {
+				continue;
+			}
+
+			$this->manifest[ $id ] = [
+										'type' => $type,
+										'herf' => $href,
+									];
+		}
+
+		return $this->manifest;
```

Comment:
Return value is redundant.

Reply:
Removed the return value. We had done it considering method chaining in future implementations if needed.

Reply:
Cool idea. You would have to return `$this` for fluent methods however. Unless I'm misunderstanding? Anyways, looks good.